src/ProjectNeon/Assets/Data/Cards/ElectroBlade/Jolt.asset

```diff
@@ -0,0 +1,112 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: 6b41b1118d12e5c48a6e7ed020b96a84, type: 3}
+  m_Name: Jolt
+  m_EditorClassIdentifier: 
+  art: {fileID: 21300000, guid: d94974f5498aa47499865518f1d2180a, type: 3}
+  description: Generate 2 Energy. Deal 1x attack. Stun.
```

Comment:
This is supposed to have a Cost of 2 Energy. The design document isn't totally clear, but that's what `- 2 Energy` means.